common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -59,7 +59,7 @@
         @if(item.content.isPaidContent && item.content.isImmersive) {
           <div class="content__section-label--advertisement">
               <a class="content__label__link content__section-label__link--advertisement" href="@LinkTo {/@item.content.sectionLabelLink}">
```

Comment:
Isn't there an option here now that we're implicitly converting to string?

Reply:
Correct, I had updated the views for `sectionLabelName` but not `sectionLabelLink`. Thanks